### DISCUSSION TITLE
Blocking All Internet Access with Exceptions in Istio

### DISCUSSION THREAD

**Discussion by Mohamed-ben-khemis** [istio/istio#53671] [Q&A]
Hello Everyone,

I’m looking for guidance on how to configure Istio to block all internet access for my services while allowing access to specific websites. Additionally, I would like to block a specific path within one of the allowed websites.

Here are the specifics of my use case:

Block all outbound internet access for my microservices, except for a few specific websites (e.g., https://example.com).
Restrict access further by blocking a specific path (e.g., example.com/registration) within the allowed website.
Could anyone provide insights or examples on how to achieve this configuration using Istio? Are there any specific resources or configurations I should look into?

Thank you for your help!

**Comment by zirain** [comment]
https://istio.io/latest/docs/tasks/traffic-management/egress/egress-gateway/ hope this is what you want.
